frontend/public/components/graphs/area.tsx

```diff
@@ -130,9 +135,12 @@ export const AreaChart: React.FC<AreaChartProps> = ({
             theme={theme}
             scale={{ x: 'time', y: 'linear' }}
             padding={padding}
-            {...(allZero && { domain: { y: [0, 1] } })}
+            domain={{
+              ...(allZero && { y: [0, 1] }),
+              ...domain,
+            }}
```

Comment:
The could be triggering unnecessary re-renders. If so, `domain` could be memoized.